.travis.yml

```diff
@@ -1,23 +1,30 @@
-language: c
+language: python
 
 sudo: false
 
 os:
     - linux
 
-env:
-  global:
-    - CONDA_DEPENDENCIES="pytest pytest-cov coverage requests flask cryptography python-dateutil"
-    - PIP_DEPENDENCIES="coveralls flask-dance pygithub humanize"
-  matrix:
-    - PYTHON_VERSION=3.6
+stages:
+    - name: Coverage
+    - name: Style
```

Comment:
Is there any benefit to using stages here?

Reply:
I was thinking a way of minimizing API limit hit. But now that I think about it, it should be "Style" first. That is, if PEP 8 is not satisfied, there is no point to do the rest. If you disagree, I can remove the stages, so just let me know.